airbyte-integrations/connectors/source-intercom/source_intercom/manifest.yaml

```diff
@@ -23,6 +23,8 @@ definitions:
     request_headers:
       Intercom-Version: '2.5' # ATTENTION: API version change is possible here
       Accept: 'application/json'
+    error_handler:
+      type: "DefaultErrorHandler"
```

Comment:
Is there a reason behind this addings?

Reply:
I was trying to use inheritance as much as possible. We have an error handler defined in parent `HttpRequester` component but it is `None` by default. So I added `DefaultErrorHandler` explicitly to the `manifest.yaml`